app/src/utils/initUtils/index.ts

```diff
@@ -96,13 +97,14 @@ export async function coreInit(): Promise<sqlite3.Database> {
   } else {
     log.info('INFO: No contacts found.');
   }
+  // TODO(Danilowicz): Can this be promise.alled ?
```

Comment:
In the current state, i don't think so since the chat table depends on the main table i think

Reply:
Yeah, makes sense